Resources/doc/menu_options.rst

```diff
@@ -0,0 +1,46 @@
+Setting Default Menu Options
+============================
+
+When creating a menu, menu options are set by passing them to ``addChild()``.
+When all menu items need the same option, adding the same options to each child
+can be cumbersome. In some cases, passing options isn't possible (e.g. when the
+menu was created based on a database).
```

Comment:
i think the last line is not needed - and a bit confusing. i can store the options in the database and have them passed.

i think the point is really avoiding redundancy or having the effort to carry around default values in your application (that you probably set up in configuration anyways). in the cmf effort its the redundancy of storing options with every node which is an even worse redundancy.